plugins/speculation-rules/tests/test-speculation-rules-wp-core-api.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * Tests for speculation-rules WP Core API.
+ *
+ * @package speculation-rules
+ */
+
+class Test_Speculation_Rules_WP_Core_API extends WP_UnitTestCase {
```

Comment:
The methods in this class should have `@covers` annotations.

Reply:
Good catch, added in https://github.com/WordPress/performance/pull/1883/commits/ef87d7a85a73176086d48ab7e06c21d5f6fd28f2